### PR TITLE
[WIP] UI refactor: folder and code structure

### PR DESCRIPTION
**Pull request by pavloko**
- [x] remove .bind and simplify state initialization
- [x] fix nested `h1` error in console
- [x] refactor dubious `index.js` component names
- [x] refactor code structure routing, etc.

### CODE REVIEW

File: ui/src/ModelsList/index.js [hunk 1]
```diff
@@ -4,14 +4,9 @@ import { ListGroupItem, ListGroup, Badge, Container } from "reactstrap";
 import { Link } from "react-router-dom";
 
 export default class ModelsList extends Component {
-  constructor(props) {
-    super(props);
+  state = { models: [], target: [] };
```
Comment (jettify) [inline ui/src/ModelsList/index.js]: this is equivalent to `constructor(props)` ?

Comment (pavloko) [reply ui/src/ModelsList/index.js]: not really equivalent, you'd still use a constructor if you need to do something when creating a class, but in this case we just need a property on the class.

File: ui/src/AggStats/index.js [hunk 2]
```diff
@@ -1,26 +1,13 @@
...
 export default class AggStats extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      success: 0,
-      error: 0,
-      mean_resp_time: 0
-    };
-    this.handleFetch = this.handleFetch.bind(this);
```
Comment (jettify) [inline ui/src/AggStats/index.js]: binding is not required anymore because of fancy babel config that enable new feature?

Comment (pavloko) [reply ui/src/AggStats/index.js]: using arrow functions solves the issue, they keep `this` binded.

**Comment by jettify** [comment]
`src` files location is convention of some sort?

File: ui/src/AggStats/index.js (hunk 2 above)
Comment (jettify) [reply ui/src/AggStats/index.js]: ah, got it

**Comment by pavloko** [comment]
hmm, changed it automatically on subconscious level, probably not really necessary :)

**Comment by pavloko** [comment]
screen shot 2018-09-23 at 3 16 40 pm

**Pull request merged**